### PR TITLE
change to prepend

### PR DESCRIPTION
**Pull request by korbzilla**
changed from append to prepend so we can see latest results

### CODE REVIEW

**Comment by sudboy123** [comment]
why

File: index.html [hunk 1]
```diff
@@ -19,12 +19,12 @@
               const result = document.createElement("div");
               result.innerHTML = `
             <h2>${o.title} ${o.id}</h2>
-            <iframe src="${o.embed_url}" width="480" height="429" frameBorder="0" class="giphy-embed" allowFullScreen></iframe>
+            <iframe src="${o.embed_url}" width="" height="429" frameBorder="0" class="giphy-embed" allowFullScreen></iframe>
```
Comment (chromemaster4756) [inline index.html]: Please don't delete the width attribute.

**Review by chromemaster4756 — CHANGES_REQUESTED** [review]

**Comment by sudboy123** [comment]
korben

**Comment by chromemaster4756** [comment]
yeet

**Review by chromemaster4756 — APPROVED** [review]

**Comment by chromemaster4756** [comment]
*claps* meme *claps* review

**Comment by sudboy123** [comment]
where? why?

**Comment by chromemaster4756** [comment]
yep

**Pull request merged**